src/core/trulens/core/utils/python.py

```diff
@@ -342,7 +343,7 @@ def wrapper(*args, **kwargs):
 STACK = "__tru_stack"
 
 
-def caller_frame(offset=0) -> frame:
+def caller_frame(offset=0) -> FrameType:
```

Comment:
PLease explain the logic chaange here.

Reply:
I fixed the return type for this method. `inspect.stack()` is a builtin python method that returns `list[FrameInfo]`. `FrameInfo.frame` would be a FrameType so this method should reflect that